yum_requirements.txt

```diff
@@ -9,3 +9,4 @@ python-netaddr
 libguestfs-tools
 bind-utils
 libselinux-python.x86_64
+libselinux-python3
```

Comment:
Does this pull in python3 itself as a dependency?  We wouldn't want that in python2 environments.

Reply:
As far as the libselinux-python3, it provides capabilities to run Selinux on python3. Download size is 222kb and does not pull python3. https://centos.pkgs.org/7/centos-aarch64/libselinux-python3-2.5-15.el7.aarch64.rpm.html